cmd/once/cmdonce/internal/machine_fingerprint.go

```diff
@@ -0,0 +1,23 @@
+package internal
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+)
+
+// GenerateMachineFingerprint generates a cryptographically secure random string
+// to represent a machine.
+// This approach works even in a containerized environment, unlike https://github.com/denisbrodbeck/machineid
+func GenerateMachineFingerprint() string {
+	// base64 encodes every 3 bytes into 4 bytes.
```

Comment:
Should we store the fingerprint somewhere in the machine?

Reply:
Yes, done in here https://github.com/authgear/authgear-server/pull/5158/commits/c85be3b4a84af49522e37b1f4c617acf98640532#diff-3b93f16b9d5f01d1c187a5d7537353531ca09fb9f5d972a33dca31f2d697515fR522-R523